apps/astro-app/src/pages/index.astro

```diff
@@ -35,6 +35,7 @@ const serverSideTitle = 'Angular (server side binding)';
 			/>
 			<CardComponent
         client:visible
+        data-analog-id="card-1"
```

Comment:
I think this would be a clearer connection

```suggestion
        data-analog-id="card-component"
```

Reply:
I agree. I also meant to illustrate that the IDs need to be unique when mapping to multiple instances, so would `card-component-1` be better suited to cover both?

Reply:
That works. Could be helpful to add a note that it should be a unique ID for each usage in the template